src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -124,6 +124,10 @@ export function updateLayeredMaterialNodeImagery(context, layer, node, parent) {
     const targetLevel = chooseNextLevelToFetch(layer.updateStrategy.type, node, destinationLevel, nodeLayer.level, layer, failureParams);
 
     if (targetLevel <= nodeLayer.level || targetLevel > destinationLevel) {
+        if (failureParams.lowestLevelError != Infinity) {
+            // this is the highest level found in case of error.
+            node.layerUpdateState[layer.id].noMoreUpdatePossible();
+        }
```

Comment:
`failureParams.lowestLevelError` can't be **inferior**  to `targetLevel` because the **next level to fetch** (`targetLevel`) returns the level between **current level** and `lowestLevelError`